core/metacling/src/TCling.h

```diff
@@ -177,6 +177,7 @@ class TCling : public TInterpreter {
    Int_t   AutoLoad(const std::type_info& typeinfo, Bool_t knowDictNotLoaded = kFALSE);
    Int_t   AutoParse(const char* cls);
    void*   LazyFunctionCreatorAutoload(const std::string& mangled_name);
+   bool   LibraryLoadingFailed(std::string, std::string, bool, bool);
```

Comment:
Since the std::string are never copied (seem to be only observed), wouldn't
```
bool   LibraryLoadingFailed(const &std::string, const &std::string, bool, bool);
```
be better (here and in the other related functions)?

Reply:
That's true, addressed